synchronizer/l1_sync_orchestration_test.go

```diff
@@ -31,8 +31,8 @@ func TestGivenOrquestrationWhenHappyPathThenReturnsBlockAndNoErrorAndProducerIsR
 	returnedBlock, err := sut.start()
 	require.NoError(t, err)
 	require.Equal(t, block, *returnedBlock)
-	require.Equal(t, true, sut.producerStarted)
```

Comment:
Why sut?? Shouldn't it be called sot??? (**S**etup**O**rchestration**T**est)

Reply:
sut stand for Subject Under Test... is a common way to name the object that you are testing